pheps/phep-0001.md

```diff
@@ -0,0 +1,287 @@
+
+# PHEP Workflow
+## PHEP Editors
+<a name="phep-editors"></a>
+The PHEP editors are individuals responsible for managing the administrative and editorial aspects of the PHEP workflow (e.g. assigning PHEP numbers and changing their status). See [PHEP Editor Responsibilities & Workflow](#phep-editor-responsibilities-workflow) for details.
+
+PHEP editorship is by nomination of the current editors, subject to approval of PyHC leadership. In the event there are fewer than two active editors, PyHC leadership may appoint editors from active participants in the community, sufficient to bring the total to two.
+
+The editors can be contacted by mentioning `@heliophysicsPy/pep-editors` on GitHub. All of the PHEP workflow is conducted via the GitHub PyHC standards repository issues and pull requests.
+
+## Start with an idea
+The PHEP process begins with a new idea for PyHC. It is highly recommended that a single PHEP contain a single key proposal or new idea; the more focused the PHEP, the more successful it tends to be. The PHEP editors reserve the right to reject PHEP proposals if they appear too unfocused or too broad. If in doubt, split your PHEP into several well-focused ones.
+
+Each PHEP must have a champion - someone who writes the PHEP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The PHEP champion (a.k.a. Author) should first attempt to ascertain whether the idea is PHEP-able. Posting to the PyHC mailing list or Element/Slack is usually the best way to go about this.
+
+Vetting an idea publicly before going as far as writing a PHEP is meant to save the potential author time. Asking the community first if an idea is original helps prevent too much time being spent on something that is guaranteed to be rejected based on prior discussions (searching the Internet does not always do the trick). It also helps to make sure the idea is applicable to the entire community and not just the author. Just because an idea sounds good to the author does not mean it will work for most people in most areas.
+
+Once the champion has asked the PyHC community as to whether an idea has any chance of acceptance, a draft PHEP should be presented to the appropriate venue mentioned above. This gives the author a chance to flesh out the draft PHEP to make it properly formatted, of high quality, and to address initial concerns about the proposal.
+
+## Submitting a PHEP
+The proposal must be submitted as a draft PHEP via a [GitHub pull request](https://github.com/heliophysicsPy/standards/pulls). The draft must be written in PHEP style as described below, else it will fail review immediately (although minor errors may be corrected by the editors).
```

Comment:
```suggestion
The proposal must be submitted as a draft PHEP via a [GitHub pull request](https://github.com/heliophysicsPy/standards/pulls). The draft must be written in PHEP style as described below before it can be officially accepted.  The editors may correct minor errors.
```

Reply:
Good change. It makes sense for something as huge as Python to insta-reject things for poor formatting, but since we're so much smaller, "before it can be officially accepted" is better language.

Although watch out for that double space you accidentally put after accepted's period @namurphy! We aren't writing on typewriters :)